fabric-ingredient-api-v1/src/main/java/net/fabricmc/fabric/api/ingredient/v1/FabricIngredient.java

```diff
@@ -0,0 +1,20 @@
+package net.fabricmc.fabric.api.ingredient.v1;
+
+import org.jetbrains.annotations.Nullable;
+
+// TODO: ignoresNbt is definitely useful, but what about the other two?
+// TODO: getCustomIngredient might return weird results for "OR" ingredients
+public interface FabricIngredient {
+	default boolean isCustom() {
+		return false;
+	}
+
+	@Nullable
+	default CustomIngredient getCustomIngredient() {
+		return null;
+	}
+
+	default boolean ignoresNbt() {
+		return true;
+	}
```

Comment:
For injected interfaces, the status quo is to not add the `fabric_` prefix. Personally. I think that it looks quite ugly, and given fabric API's privileged status we can request other mods to adjust their methods in case of a name conflict.